prompting/base/prompting_miner.py

```diff
@@ -23,7 +23,7 @@
 from prompting.protocol import StreamPromptingSynapse
 from prompting.base.miner import BaseStreamMinerNeuron
 from datetime import datetime
-
+from typing import List, Dict
```

Comment:
for future reference: it's best to use built-in types over typing since introduction of built-in typehints

Reply:
Yeah this is a holdover from when we supported 3.8